modules/network_synchronizer/bit_array.cpp

```diff
@@ -90,8 +90,6 @@ void BitArray::store_bits(int p_bit_offset, uint64_t p_value, int p_bits) {
 
 		val >>= bits_to_write;
 	}
-
-	CRASH_COND_MSG(val != 0, "This is a bug, please open an issue");
```

Comment:
Was this been removed for mistake?

Reply:
Yes, sorry :)

Reply:
Oh, no, this causes issues when I sending signed values with shifted. I would have to manually zero out the unnecessary values. I thought that the check could be removed, since we have already covered everything with tests.

Reply:
Yup, that's fine, you right. The test are much better than this check.